spyder/app/tour.py

```diff
@@ -144,68 +144,74 @@ def get_tour(index):
+                           "Here, we are going to guide you through its most "
                            "important features.<br><br>"
                            "Please use the arrow keys or click on the buttons "
                            "below to move along the tour."),
               'image': 'tour-spyder-logo.png'},
 
              {'title': _("The Editor"),
-              'content': _("This is the pane where you write Python code before "
-                           "evaluating it. You can get automatic suggestions "
-                           "and completions while writing, by pressing the "
-                           "<b>Tab</b> key next to a given text.<br><br>"
-                           "The Editor comes "
-                           "with a line number area (highlighted here in red), "
-                           "where Spyder shows warnings and syntax errors. They "
-                           "can help you to detect potential problems before "
-                           "running the code.<br><br>"
+              'content': _("This is where you write Python code before "
+                           "evaluating it. You can get automatic "
+                           "completions while typing by pressing the "
+                           "<b>Tab</b> key, or automatically by enabling "
+                           "on-the-fly completions in Spyder's Preferences. "
+                           "<br><br>The Editor comes "
+                           "with a line number area (highlighted here in red) "
+                           "where Spyder shows warnings and syntax errors. "
+                           "They can help you to detect potential problems "
+                           "before running your code.<br><br>"
                            "You can also set debug breakpoints in the line "
-                           "number area, by doing a double click next to "
-                           "a non-empty line."),
+                           "number area by clicking next to a "
+                           "any non-empty line."),
               'widgets': [sw.editor],
               'decoration': [sw.editor_line_number_area]},
 
-             {'title': _("The IPython console"),
-              'content': _("This is one of panes where you can run or "
-                           "execute the code you wrote on the Editor. To do it "
-                           "you need to press the <b>F5</b> key.<br><br>"
-                           "This console comes with several "
+             {'title': _("IPython Console"),
+              'content': _("This is where you can run Python code, either "
+                           "from the Editor or interactively. To run the "
+                           "current file, press <b>F5</b> by default, "
+                           "or press <b>Ctrl+Enter</b> "
+                           "(<b>Cmd-Return</b> on Mac) to execute the current "
+                           "line or selection.<br><br>"

```

Comment:
This doesn't work in Spyder 4 (it worked in version 3 though): the way to run the current line or selection is by pressing `F9`.

`Ctrl/Cmd + Enter` is used exclusively to run cells now (unless you remap that shortcut to run the current line/selection).

Reply:
Sorry, on my stable copy I've had the various run shortcuts remapped to be Enter + modifier keys, rather than the default of some being F-keys and some being Enter + Modifiers that doesn't make any sense to me and is harder to press. I thought I'd referenced Spyder running in safe mode to check the default, as I usually do, but evidently I accidentally checked my stable copy instead, my mistake. I'll fix that.